resources/js/app.js

```diff
@@ -7,14 +7,16 @@ import { InertiaApp } from '@inertiajs/inertia-vue';
 import { InertiaForm } from 'laravel-jetstream';
 import PortalVue from 'portal-vue';
 import Calendar from 'v-calendar/lib/components/calendar.umd'
-import DatePicker from 'v-calendar/lib/components/date-picker.umd'
 import DateFormatter from "@src/Plugins/date-formatter";
 
+import VueCtkDateTimePicker from 'vue-ctk-date-time-picker/dist/vue-ctk-date-time-picker.umd';
+import 'vue-ctk-date-time-picker/dist/vue-ctk-date-time-picker.css';
```

Comment:
Including CSS like this will work? TIL

Reply:
Ye, definitely odd. comes from install instructions: https://github.com/chronotruck/vue-ctk-date-time-picker

Also if I don't add this here but try to import in the DateTimePicker.vue file, Vue will throw warnings about infinite loops and the inputs won't work 🤷

Reply:
Interesting, and if you include it in app.css instead of here it also throws warnings?

Reply:
nah css should be fine. It's the component import that do the infinite loop thing. something weird with UMD modules, ES6 all the way